### DISCUSSION TITLE
Running Docker

### DISCUSSION THREAD

**Discussion by mwamimwiza** [CyferShepard/Jellystat#110] [Q&A]
![Capture](https://github.com/CyferShepard/Jellystat/assets/23127939/15219e1e-fd80-4c87-8357-32d8a4007f4a)

Hello, 
I have tried to run the docker image but I am running into trouble telling me it can't find the Postgres configurations.
is it possible to have installation instructions?  i am pulling the docker image,then run with "docker run -dp 127.0.0.1:3000:3000 cyfershepard/jellystat" .when i run the browser i get this as message "Error: Unable to connect to Jellystat Backend". 

inside the console, i am getting this output
"
[0] Error: Postgres details not defined
[0] JWT Secret cannot be undefined
[0] npm run start-server exited with code 1

"

**Comment by CyferShepard** [comment]
Hi @mwamimwiza , your command is missing the env variables, you can check the docker compose file for reference
